### PR TITLE
Add licenses & more

### PR DESCRIPTION
**Pull request by gzussa**

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+# Rules are matched bottom-to-top, so one team can own subdirectories
+# and another team can own the rest of the directory.
+
+# All your base
+*                       @DataDog/integrations-tools-and-libraries
+
+# Documentation
+*.md                    @DataDog/baklava
+LICENSE                 @DataDog/baklava
+NOTICE                  @DataDog/baklava
```
Comment (jeremy-lq) [inline .github/CODEOWNERS]: NOTICE file is only needed for Apache 2

File: ICENSE-3rdparty.csv [hunk 2]
```diff
@@ -0,0 +1 @@
+Component,Origin,License,Copyright
```
Comment (jeremy-lq) [inline ICENSE-3rdparty.csv]: Filename is missing the leading L.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+  * `changelog/Changed` for changes in existing functionality.
+  * `changelog/Deprecated` for soon-to-be removed features.
+  * `changelog/Removed` for now removed features.
+  * `changelog/Fixed` for any bug fixes.
+  * `changelog/Security` in case of vulnerabilities.
+  * `changelog/no-changelog` in case this PR should not appear in the changelog at all.
+
+See [here][4] for more details about changelogs.
+
+Your pull request must pass all CI tests before we will merge it. If you're seeing
```
Comment (zippolyte) [inline CONTRIBUTING.md]:
```suggestion
Your pull request must pass all CI tests before. If you're seeing
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+an error and don't think it's your fault, it may not be! [Join us on Slack][5] or send us an email, and together we'll 
+get it sorted out.
+
+### Keep it small, focused
+
+Avoid changing too many things at once. For instance if you're fixing two different
+checks at once, it makes reviewing harder and the _time-to-release_ longer.
```
Comment (zippolyte) [inline CONTRIBUTING.md]:
```suggestion
issues at once, it makes reviewing harder and the _time-to-release_ longer.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+| `documentation`               | Issues and Pull Requests | Documentation related changes.
+| `duplicate`                   | Issue Only               | Duplicate issue.
+| `invalid`                     | Issue Only               | Invalid issue.
+| `kind/bug`                    | Issue Only               | Bug related issue.
+| `kind/feature-request`        | Issue Only               | Feature request related issue.
+| `severity/critical`           | Issue Only               | Critical severity issue.
+| `severity/major`              | Issue Only               | Major severity issue.
+| `severity/minor`              | Issue Only               | Minor severity issue.
+| `severity/normal`             | Issue Only               | Normal severity issue.
+| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
```
Comment (zippolyte) [inline CONTRIBUTING.md]:
```suggestion
| `stale`                       | Issues and Pull Requests | Stale - Bot reminder.
| `stale/exempt`                | Issues and Pull Requests | Exempt from being marked as stale.

```

File: DEVELOPMENT.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
+# Development
+
+// TODO
```
Comment (zippolyte) [inline DEVELOPMENT.md]: let's fill TODOs or remove

File: RELEASING.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+## Schedule
+This project does not have a strict release schedule. However, we would make a release at least every 2 months.
+  - No release will be done if no changes got merged to the `master` branch during the above mentioned window.
+  - Releases may be done more frequently than the above mentioned window.
+  - Create a pager duty schedule corresponding to this.
+  - Create a google calendar schedule corresponding to this.
+
+## Make Sure Everything Works
+
+// TODO
```
Comment (zippolyte) [inline RELEASING.md]: Fill TODO

File: RELEASING.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+### Commands
+
+- See changes ready for release by running `ddev release show changes .` at the root of this project. Add any missing labels to PRs if needed.
+- Run `ddev release changelog . <NEW_VERSION>` to update the `CHANGELOG.md` file at the root of this repository
+- Commit the changes to the repository in a release branch and get it approved/merged.
+- Tag the repository with the new version number.
+
+## Release
+
+- Update the [CHANGELOG](#commands) and [pimp it](https://github.com/pcreux/pimpmychangelog).
```
Comment (zippolyte) [inline RELEASING.md]: outdated

File: spec/unit/facade_spec.rb [hunk 9]
```diff
@@ -1,4 +1,8 @@
-require 'spec_helper'
+# Unless explicitly stated otherwise all files in this repository are licensed under the BSD-3-Clause License.
+# This product includes software developed at Datadog (https://www.datadoghq.com/).
+# Copyright 2011-Present Datadog, Inc.
+
+equire 'spec_helper'
```
Comment (zippolyte) [inline spec/unit/facade_spec.rb]:
```suggestion
require 'spec_helper'
```

**Pull request merged**